From 04b14c8fac60a0904066af2ae66df0c409164c10 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Wed, 15 Feb 2023 18:05:43 +0000
Subject: [PATCH] hwdb: update

(cherry picked from commit 0c1f5d50be2d7a99ad4259eb1486724e17af6964)

Related: RHEL-5950
---
 hwdb.d/20-OUI.hwdb               |  90 +++++-
 hwdb.d/20-acpi-vendor.hwdb.patch |   4 +-
 hwdb.d/20-pci-vendor-model.hwdb  | 518 +++++++++++++++++++++++++------
 hwdb.d/ma-large.txt              | 147 +++++++--
 hwdb.d/ma-medium.txt             |  56 +++-
 hwdb.d/ma-small.txt              |  57 +++-
 hwdb.d/pci.ids                   | 136 +++++++-
 7 files changed, 847 insertions(+), 161 deletions(-)

diff --git a/hwdb.d/20-OUI.hwdb b/hwdb.d/20-OUI.hwdb
index 3836339af0..69b00c94c3 100644
--- a/hwdb.d/20-OUI.hwdb
+++ b/hwdb.d/20-OUI.hwdb
@@ -43718,6 +43718,9 @@ OUI:101331*
 OUI:1013EE*
  ID_OUI_FROM_DATABASE=Justec International Technology INC.
 
+OUI:1015C1*
+ ID_OUI_FROM_DATABASE=Zhanzuo (Beijing) Technology Co., Ltd.
+
 OUI:10189E*
  ID_OUI_FROM_DATABASE=Elmo Motion Control
 
@@ -48060,7 +48063,7 @@ OUI:1CA0D36*
  ID_OUI_FROM_DATABASE=Intertecno SRL NISUTA
 
 OUI:1CA0D37*
- ID_OUI_FROM_DATABASE=Private
+ ID_OUI_FROM_DATABASE=U-TX Technologies Ltd
 
 OUI:1CA0D38*
  ID_OUI_FROM_DATABASE=Desarrollos y Soluciones Guinea I+D S.L.
@@ -50448,7 +50451,7 @@ OUI:24BE18*
  ID_OUI_FROM_DATABASE=DADOUTEK COMPANY LIMITED
 
 OUI:24BF74*
- ID_OUI_FROM_DATABASE=Private
+ ID_OUI_FROM_DATABASE=Hamamatsu Photonics K.K.
 
 OUI:24C0B3*
  ID_OUI_FROM_DATABASE=RSF
@@ -57893,6 +57896,9 @@ OUI:3CDFA9*
 OUI:3CDFBD*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:3CE002*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:3CE038*
  ID_OUI_FROM_DATABASE=Omnifi Inc.
 
@@ -68351,6 +68357,9 @@ OUI:641A22*
 OUI:641ABA*
  ID_OUI_FROM_DATABASE=Dryad Networks GmbH
 
+OUI:641C10*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:641C67*
  ID_OUI_FROM_DATABASE=DIGIBRAS INDUSTRIA DO BRASILS/A
 
@@ -71486,21 +71495,39 @@ OUI:7006920*
 OUI:7006921*
  ID_OUI_FROM_DATABASE=Beijing Fortech Microsystems., Co., Ltd.
 
+OUI:7006922*
+ ID_OUI_FROM_DATABASE=Scud (Fujian) Electronics Co.,Ltd
+
 OUI:7006923*
  ID_OUI_FROM_DATABASE=BOSSCCTV CO., LTD
 
 OUI:7006924*
  ID_OUI_FROM_DATABASE=Fusiostor Technologies Private Limited
 
+OUI:7006925*
+ ID_OUI_FROM_DATABASE=CANAAN CREATIVE CO.,LTD.
+
+OUI:7006926*
+ ID_OUI_FROM_DATABASE=Hangzhou Clounix Technology Limited
+
+OUI:7006928*
+ ID_OUI_FROM_DATABASE=JMA Wireless
+
 OUI:7006929*
  ID_OUI_FROM_DATABASE=Shenzhen Lingwei Technology Co., Ltd
 
 OUI:700692A*
  ID_OUI_FROM_DATABASE=Munters
 
+OUI:700692B*
+ ID_OUI_FROM_DATABASE=SWIT Electronics Co.,Ltd
+
 OUI:700692C*
  ID_OUI_FROM_DATABASE=ScoreBird, LLC
 
+OUI:700692D*
+ ID_OUI_FROM_DATABASE=Skyware Protech Limited
+
 OUI:7006AC*
  ID_OUI_FROM_DATABASE=Eastcompeace Technology Co., Ltd
 
@@ -77739,7 +77766,7 @@ OUI:70B3D56F3*
  ID_OUI_FROM_DATABASE=iungo
 
 OUI:70B3D56F4*
- ID_OUI_FROM_DATABASE=Private
+ ID_OUI_FROM_DATABASE=WDI Wise Device Inc.
 
 OUI:70B3D56F5*
  ID_OUI_FROM_DATABASE=Cominfo, Inc.
@@ -85754,6 +85781,9 @@ OUI:74A4B5*
 OUI:74A528*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
+OUI:74A58C*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:74A6CD*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
@@ -90482,6 +90512,9 @@ OUI:84716A*
 OUI:847207*
  ID_OUI_FROM_DATABASE=I&C Technology
 
+OUI:847293*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:847303*
  ID_OUI_FROM_DATABASE=Letv Mobile and Intelligent Information Technology (Beijing) Corporation Ltd.
 
@@ -91154,6 +91187,9 @@ OUI:84F129*
 OUI:84F147*
  ID_OUI_FROM_DATABASE=Cisco Systems, Inc
 
+OUI:84F175*
+ ID_OUI_FROM_DATABASE=Jiangxi Xunte Intelligent Terminal Co., Ltd
+
 OUI:84F1D0*
  ID_OUI_FROM_DATABASE=EHOOME IOT PRIVATE LIMITED
 
@@ -91463,6 +91499,9 @@ OUI:883D24*
 OUI:883F0C*
  ID_OUI_FROM_DATABASE=system a.v. co., ltd.
 
+OUI:883F37*
+ ID_OUI_FROM_DATABASE=UHTEK CO., LTD.
+
 OUI:883F4A*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
@@ -92765,6 +92804,9 @@ OUI:8C1F640D5*
 OUI:8C1F640D6*
  ID_OUI_FROM_DATABASE=AVD INNOVATION LIMITED
 
+OUI:8C1F640D8*
+ ID_OUI_FROM_DATABASE=Power Electronics Espana, S.L.
+
 OUI:8C1F640E0*
  ID_OUI_FROM_DATABASE=Autopharma
 
@@ -93167,6 +93209,9 @@ OUI:8C1F64317*
 OUI:8C1F6431A*
  ID_OUI_FROM_DATABASE=Asiga Pty Ltd
 
+OUI:8C1F6431B*
+ ID_OUI_FROM_DATABASE=joint analytical systems GmbH
+
 OUI:8C1F64324*
  ID_OUI_FROM_DATABASE=Kinetic Technologies
 
@@ -95240,6 +95285,9 @@ OUI:8C1F64DC9*
 OUI:8C1F64DCA*
  ID_OUI_FROM_DATABASE=Porsche engineering
 
+OUI:8C1F64DD4*
+ ID_OUI_FROM_DATABASE=Midlands Technical Co., Ltd.
+
 OUI:8C1F64DD5*
  ID_OUI_FROM_DATABASE=Cardinal Scales Manufacturing Co
 
@@ -95276,6 +95324,9 @@ OUI:8C1F64E12*
 OUI:8C1F64E21*
  ID_OUI_FROM_DATABASE=LG-LHT Aircraft Solutions GmbH
 
+OUI:8C1F64E24*
+ ID_OUI_FROM_DATABASE=COMETA SAS
+
 OUI:8C1F64E2D*
  ID_OUI_FROM_DATABASE=Private
 
@@ -96653,6 +96704,9 @@ OUI:8CF228*
 OUI:8CF319*
  ID_OUI_FROM_DATABASE=Siemens Industrial Automation Products Ltd., Chengdu
 
+OUI:8CF3E7*
+ ID_OUI_FROM_DATABASE=solidotech
+
 OUI:8CF5A3*
  ID_OUI_FROM_DATABASE=SAMSUNG ELECTRO-MECHANICS(THAILAND)
 
@@ -96746,6 +96800,9 @@ OUI:9003B7*
 OUI:900628*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:9006F2*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:900917*
  ID_OUI_FROM_DATABASE=Far-sighted mobile
 
@@ -99068,6 +99125,9 @@ OUI:9802D8E*
 OUI:9802D8F*
  ID_OUI_FROM_DATABASE=Private
 
+OUI:98038A*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:98039B*
  ID_OUI_FROM_DATABASE=Mellanox Technologies, Inc.
 
@@ -102002,6 +102062,9 @@ OUI:A06518*
 OUI:A06610*
  ID_OUI_FROM_DATABASE=FUJITSU LIMITED
 
+OUI:A06636*
+ ID_OUI_FROM_DATABASE=Intracom SA Telecom Solutions
+
 OUI:A06720*
  ID_OUI_FROM_DATABASE=China Dragon Technology Limited
 
@@ -114704,6 +114767,9 @@ OUI:CCB3AB*
 OUI:CCB3F8*
  ID_OUI_FROM_DATABASE=FUJITSU ISOTEC LIMITED
 
+OUI:CCB54C*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:CCB55A*
  ID_OUI_FROM_DATABASE=Fraunhofer ITWM
 
@@ -116801,6 +116867,9 @@ OUI:D45251*
 OUI:D45297*
  ID_OUI_FROM_DATABASE=nSTREAMS Technologies, Inc.
 
+OUI:D452C7*
+ ID_OUI_FROM_DATABASE=Beijing L&S Lancom Platform Tech. Co., Ltd.
+
 OUI:D452EE*
  ID_OUI_FROM_DATABASE=SKY UK LIMITED
 
@@ -117554,6 +117623,9 @@ OUI:D4E8B2*
 OUI:D4E90B*
  ID_OUI_FROM_DATABASE=CVT CO.,LTD
 
+OUI:D4E95E*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:D4E98A*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -126020,6 +126092,9 @@ OUI:F4A80D*
 OUI:F4A997*
  ID_OUI_FROM_DATABASE=CANON INC.
 
+OUI:F4AAD0*
+ ID_OUI_FROM_DATABASE=OHSUNG
+
 OUI:F4ACC1*
  ID_OUI_FROM_DATABASE=Cisco Systems, Inc
 
@@ -126650,6 +126725,9 @@ OUI:F82D7C*
 OUI:F82DC0*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
+OUI:F82E0C*
+ ID_OUI_FROM_DATABASE=Texas Instruments
+
 OUI:F82E3F*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -127934,6 +128012,9 @@ OUI:FC45C3*
 OUI:FC47D8*
  ID_OUI_FROM_DATABASE=Apple, Inc.
 
+OUI:FC48C9*
+ ID_OUI_FROM_DATABASE=Yobiiq Intelligence B.V.
+
 OUI:FC48EF*
  ID_OUI_FROM_DATABASE=HUAWEI TECHNOLOGIES CO.,LTD
 
@@ -128177,6 +128258,9 @@ OUI:FC83C6*
 OUI:FC8417*
  ID_OUI_FROM_DATABASE=Honor Device Co., Ltd.
 
+OUI:FC84A7*
+ ID_OUI_FROM_DATABASE=Murata Manufacturing Co., Ltd.
+
 OUI:FC8596*
  ID_OUI_FROM_DATABASE=Axonne Inc.
 
diff --git a/hwdb.d/20-acpi-vendor.hwdb.patch b/hwdb.d/20-acpi-vendor.hwdb.patch
index 0fb305a9f9..6d4e61c8eb 100644
--- a/hwdb.d/20-acpi-vendor.hwdb.patch
+++ b/hwdb.d/20-acpi-vendor.hwdb.patch
@@ -1,5 +1,5 @@
---- 20-acpi-vendor.hwdb.base	2023-02-10 16:14:35.983480917 +0000
-+++ 20-acpi-vendor.hwdb	2023-02-10 16:14:35.991481074 +0000
+--- 20-acpi-vendor.hwdb.base	2023-02-15 18:03:45.791408751 +0000
++++ 20-acpi-vendor.hwdb	2023-02-15 18:03:45.799408909 +0000
 @@ -3,6 +3,8 @@
  # Data imported from:
  #     https://uefi.org/uefi-pnp-export
diff --git a/hwdb.d/20-pci-vendor-model.hwdb b/hwdb.d/20-pci-vendor-model.hwdb
index eba428890c..c43385d4b9 100644
--- a/hwdb.d/20-pci-vendor-model.hwdb
+++ b/hwdb.d/20-pci-vendor-model.hwdb
@@ -11793,79 +11793,79 @@ pci:v00001002d00009552sv0000174Bsd00003000*
  ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4330/4350/4550] (Radeon HD 4350/4550 HyperMemory DDR2)
 
 pci:v00001002d00009553*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v]
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v]
 
 pci:v00001002d00009553sv00001025sd0000015E*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001025sd0000017D*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001025sd00000205*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570 / 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570 / 545v)
 
 pci:v00001002d00009553sv00001025sd00000206*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001025sd00000237*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001028sd000002BE*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570 / 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570 / 545v)
 
 pci:v00001002d00009553sv00001028sd000002E8*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4530)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4530)
 
 pci:v00001002d00009553sv0000103Csd0000143C*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv0000103Csd00001446*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv0000103Csd00003624*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4530)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4530)
 
 pci:v00001002d00009553sv0000103Csd00003628*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4530)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4530)
 
 pci:v00001002d00009553sv0000103Csd00003636*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4530)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4530)
 
 pci:v00001002d00009553sv00001043sd00001B32*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001043sd00001B42*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv0000104Dsd00009056*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 4570)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 4570)
 
 pci:v00001002d00009553sv00001179sd0000FF82*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Satellite L505-13T GPU (Mobility Radeon HD 5145))
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Satellite L505-13T GPU (Mobility Radeon HD 5145))
 
 pci:v00001002d00009553sv0000144Dsd0000C07F*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv0000144Dsd0000C571*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv00001462sd00001006*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv000017AAsd00002129*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv000017AAsd0000215B*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009553sv000017AAsd000021BB*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/545v] (Mobility Radeon HD 545v)
+ ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v] (Mobility Radeon HD 545v)
 
 pci:v00001002d00009555*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4350/4550]
+ ID_MODEL_FROM_DATABASE=RV711/M93 [Mobility Radeon HD 4350/4550/530v/540v/545v / FirePro RG220]
 
 pci:v00001002d00009555sv0000103Csd00001411*
- ID_MODEL_FROM_DATABASE=RV710/M92 [Mobility Radeon HD 4350/4550] (ProBook 4720s GPU (Mobility Radeon HD 4350))
+ ID_MODEL_FROM_DATABASE=RV711/M93 [Mobility Radeon HD 4350/4550/530v/540v/545v / FirePro RG220] (ProBook 4720s GPU (Mobility Radeon HD 4350))
 
 pci:v00001002d00009557*
  ID_MODEL_FROM_DATABASE=RV711/M93 GL [FirePro RG220]
@@ -37685,6 +37685,9 @@ pci:v000010DEd000025A9*
 pci:v000010DEd000025AA*
  ID_MODEL_FROM_DATABASE=GA107M [GeForce MX570 A]
 
+pci:v000010DEd000025AB*
+ ID_MODEL_FROM_DATABASE=GA107M [GeForce RTX 3050 4GB Laptop GPU]
+
 pci:v000010DEd000025AC*
  ID_MODEL_FROM_DATABASE=GN20-P0-R-K2 [GeForce RTX 3050 6GB Laptop GPU]
 
@@ -37712,6 +37715,9 @@ pci:v000010DEd000025BA*
 pci:v000010DEd000025BB*
  ID_MODEL_FROM_DATABASE=GA107GLM [RTX A500 Laptop GPU]
 
+pci:v000010DEd000025BC*
+ ID_MODEL_FROM_DATABASE=GA107
+
 pci:v000010DEd000025E0*
  ID_MODEL_FROM_DATABASE=GA107BM [GeForce RTX 3050 Ti Mobile]
 
@@ -37760,6 +37766,9 @@ pci:v000010DEd00002704*
 pci:v000010DEd00002717*
  ID_MODEL_FROM_DATABASE=GN21-X11 [GeForce RTX 4090 Laptop GPU]
 
+pci:v000010DEd00002730*
+ ID_MODEL_FROM_DATABASE=AD103GLM [RTX 5000 Ada Generation Laptop GPU]
+
 pci:v000010DEd00002757*
  ID_MODEL_FROM_DATABASE=GN21-X11
 
@@ -37778,12 +37787,21 @@ pci:v000010DEd000027A0*
 pci:v000010DEd000027B8*
  ID_MODEL_FROM_DATABASE=AD104GL [L4]
 
+pci:v000010DEd000027BA*
+ ID_MODEL_FROM_DATABASE=AD104
+
+pci:v000010DEd000027BB*
+ ID_MODEL_FROM_DATABASE=AD104GLM [RTX 3500 Ada Generation Laptop GPU]
+
 pci:v000010DEd000027E0*
  ID_MODEL_FROM_DATABASE=GN21-X9
 
 pci:v000010DEd00002820*
  ID_MODEL_FROM_DATABASE=GN21-X6
 
+pci:v000010DEd00002838*
+ ID_MODEL_FROM_DATABASE=AD106GLM [RTX 3000 Ada Generation Laptop GPU]
+
 pci:v000010DEd00002860*
  ID_MODEL_FROM_DATABASE=GN21-X6
 
@@ -37793,6 +37811,9 @@ pci:v000010DEd000028A0*
 pci:v000010DEd000028A1*
  ID_MODEL_FROM_DATABASE=GN21-X2
 
+pci:v000010DEd000028B8*
+ ID_MODEL_FROM_DATABASE=AD107GLM [RTX 2000 Ada Generation Laptop GPU]
+
 pci:v000010DEd000028E0*
  ID_MODEL_FROM_DATABASE=GN21-X4
 
@@ -41076,22 +41097,22 @@ pci:v00001106d00003218*
  ID_MODEL_FROM_DATABASE=K8T800M Host Bridge
 
 pci:v00001106d00003227*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South]
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South]
 
 pci:v00001106d00003227sv00001043sd000080ED*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South] (A7V600/K8V-X/A8V Deluxe motherboard)
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South] (A7V600/K8V-X/A8V Deluxe motherboard)
 
 pci:v00001106d00003227sv00001106sd00003227*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South] (DFI KT600-AL / Soltek SL-B9D-FGR Motherboard)
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South] (VT8237 ISA bridge)
 
 pci:v00001106d00003227sv00001458sd00005001*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South] (GA-7VT600 Motherboard)
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South] (GA-7VT600 Motherboard)
 
 pci:v00001106d00003227sv0000147Bsd00001407*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South] (KV8-MAX3 motherboard)
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South] (KV8-MAX3 motherboard)
 
 pci:v00001106d00003227sv00001849sd00003227*
- ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890 South] (K7VT4 motherboard)
+ ID_MODEL_FROM_DATABASE=VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South] (K7VT4 motherboard)
 
 pci:v00001106d00003230*
  ID_MODEL_FROM_DATABASE=K8M890CE/K8N890CE [Chrome 9]
@@ -56849,6 +56870,12 @@ pci:v0000144Dd00001600*
 pci:v0000144Dd0000A544*
  ID_MODEL_FROM_DATABASE=Exynos 8890 PCIe Root Complex
 
+pci:v0000144Dd0000A575*
+ ID_MODEL_FROM_DATABASE=Exynos 7420 PCIe Root Complex
+
+pci:v0000144Dd0000A5E3*
+ ID_MODEL_FROM_DATABASE=Exynos 5433 PCIe Root Complex
+
 pci:v0000144Dd0000A800*
  ID_MODEL_FROM_DATABASE=XP941 PCIe SSD
 
@@ -65327,9 +65354,21 @@ pci:v000016E2d00001100*
 pci:v000016E2d00001120*
  ID_MODEL_FROM_DATABASE=GX1120 Arbitrary Waveform and Function Generator PXI Board
 
+pci:v000016E2d00001164*
+ ID_MODEL_FROM_DATABASE=GX1164 Multi-Channel Programmable Resistor PXI Board
+
 pci:v000016E2d00001632*
  ID_MODEL_FROM_DATABASE=GX1632e 32-Channel Arbitrary Analog Output PXIe Board
 
+pci:v000016E2d00001648*
+ ID_MODEL_FROM_DATABASE=GX1648 64-Channel Analog Output PXI Board
+
+pci:v000016E2d00001649*
+ ID_MODEL_FROM_DATABASE=GX1649 64-Channel Arbitrary Analog Output PXI Board
+
+pci:v000016E2d00001838*
+ ID_MODEL_FROM_DATABASE=GX1838 Precision DC Source PXI Board
+
 pci:v000016E2d00002065*
  ID_MODEL_FROM_DATABASE=GX2065 Digital Multimeter PXI Board
 
@@ -65339,12 +65378,150 @@ pci:v000016E2d00002200*
 pci:v000016E2d00002201*
  ID_MODEL_FROM_DATABASE=GC2200 High Resolution Universal Time Interval Counter PCI Board
 
+pci:v000016E2d00003104*
+ ID_MODEL_FROM_DATABASE=GX3104 4-Channel 20V, 250mA Per Channel SMU PXI Board
+
+pci:v000016E2d00003108*
+ ID_MODEL_FROM_DATABASE=GX3108 4-Channel Source Measure Unit PXI Board
+
+pci:v000016E2d00003116*
+ ID_MODEL_FROM_DATABASE=GX3116e 16-Channel DUT Power Supply PXIe Board
+
+pci:v000016E2d00003348*
+ ID_MODEL_FROM_DATABASE=GX3348 Multi-Channel DC Source and Switch Matrix Board
+
+pci:v000016E2d00003500*
+ ID_MODEL_FROM_DATABASE=GX3500 Digital I/O FLEX FPGA Board
+
+pci:v000016E2d00003700*
+ ID_MODEL_FROM_DATABASE=GX3700e Digital I/O FLEX FPGA PXIe Board
+
+pci:v000016E2d00003701*
+ ID_MODEL_FROM_DATABASE=GX3700 Digital I/O FLEX FPGA PXI Board
+
+pci:v000016E2d00003800*
+ ID_MODEL_FROM_DATABASE=GX3800e Digital I/O FLEX FPGA PXIe Board
+
+pci:v000016E2d00004943*
+ ID_MODEL_FROM_DATABASE=GX4943 HellFire/Longbow Serial Communication Board
+
 pci:v000016E2d00005050*
  ID_MODEL_FROM_DATABASE=GC5050 Dynamic Digital I/O with Algorithmic Sequencer PCI Board
 
+pci:v000016E2d00005055*
+ ID_MODEL_FROM_DATABASE=GX5055 Dynamic Digital I/O with Pin Electronics PXI Board
+
+pci:v000016E2d00005083*
+ ID_MODEL_FROM_DATABASE=GX5083 Dynamic Digital I/O with Algorithmic Sequencer PXI Board
+
+pci:v000016E2d00005106*
+ ID_MODEL_FROM_DATABASE=GX5106 High-Speed Digital I/O Cycle Timing (Master) Module PXI Board
+
+pci:v000016E2d00005151*
+ ID_MODEL_FROM_DATABASE=GX5150/1 High Speed Digital I/O (50/100MHz) PXI Board
+
+pci:v000016E2d00005152*
+ ID_MODEL_FROM_DATABASE=GX5152/3 DSR High Speed Digital Stimulus/Response PXI Board
+
+pci:v000016E2d00005250*
+ ID_MODEL_FROM_DATABASE=GX5250 Digital I/O Streaming PXI Board
+
+pci:v000016E2d00005280*
+ ID_MODEL_FROM_DATABASE=GX5280 High-Speed Digital I/O PXI Board
+
+pci:v000016E2d00005290*
+ ID_MODEL_FROM_DATABASE=GX5290 High-Speed Dynamic Digital I/O PXI Board
+
+pci:v000016E2d00005294*
+ ID_MODEL_FROM_DATABASE=GX5294 High-Speed Digital I/O PXI Board
+
+pci:v000016E2d00005295*
+ ID_MODEL_FROM_DATABASE=GX5295 Digital I/O with Pin Electronics and PMU PXI Board
+
+pci:v000016E2d00005296*
+ ID_MODEL_FROM_DATABASE=GX5296 Digital I/O with Pin Electronics and PMU PXI Board
+
+pci:v000016E2d00005550*
+ ID_MODEL_FROM_DATABASE=GX5050 Dynamic Digital I/O with Algorithmic Sequencer PXI Board
+
 pci:v000016E2d00005641*
  ID_MODEL_FROM_DATABASE=GX5641/2 Bi-directional Differential-TTL I/O PXI Board
 
+pci:v000016E2d00005731*
+ ID_MODEL_FROM_DATABASE=GX5731 Digital I/O with 128 TTL Channels and 3x32 Customizable Channels PXI Board
+
+pci:v000016E2d00005732*
+ ID_MODEL_FROM_DATABASE=GX5732 224-Channel Static Digital I/O PXI Board
+
+pci:v000016E2d00005733*
+ ID_MODEL_FROM_DATABASE=GX5733 Digital I/O with 96 TTL Channels and 32 Customizable Channels PXI Board
+
+pci:v000016E2d00005960*
+ ID_MODEL_FROM_DATABASE=GX5961/4 Digital I/O Timing/Sync with Pin Electronics and PMU PXI Board
+
+pci:v000016E2d00006021*
+ ID_MODEL_FROM_DATABASE=GX6021 20-Channel RF Multiplexer PXI Board
+
+pci:v000016E2d00006062*
+ ID_MODEL_FROM_DATABASE=GX6062 60-Channel RF Multiplexer PXI Board
+
+pci:v000016E2d00006115*
+ ID_MODEL_FROM_DATABASE=GX6115 15-Channel High Current SPDT Form-C Relay PXI Board
+
+pci:v000016E2d00006125*
+ ID_MODEL_FROM_DATABASE=GX6125 25-Channel High-Density SPDT Form-C Relay PXI Board
+
+pci:v000016E2d00006138*
+ ID_MODEL_FROM_DATABASE=GX6138 38-Channel SPST Form-A Relay PXI Board
+
+pci:v000016E2d00006188*
+ ID_MODEL_FROM_DATABASE=GX6188 104-Channel 8x8 Multiplexer/Matrix PXI Board
+
+pci:v000016E2d00006192*
+ ID_MODEL_FROM_DATABASE=GX6192 16, 16x2 High Frequency, High Density Multiplexer PXI Board
+
+pci:v000016E2d00006196*
+ ID_MODEL_FROM_DATABASE=GX6196 96-Channel DPST Form-A Relay Carrier PXI Board
+
+pci:v000016E2d00006256*
+ ID_MODEL_FROM_DATABASE=GX6256 16, 16x2 Low Frequency, High Density Multiplexer PXI Board
+
+pci:v000016E2d00006264*
+ ID_MODEL_FROM_DATABASE=GX6264 128-Channel Scanner/Multiplexer Board
+
+pci:v000016E2d00006315*
+ ID_MODEL_FROM_DATABASE=GX6315 45-Channel High Current SPDT Form-C Relay Board
+
+pci:v000016E2d00006325*
+ ID_MODEL_FROM_DATABASE=GX6325 75-Channel High-Density SPDT Form-C Relay Board
+
+pci:v000016E2d00006338*
+ ID_MODEL_FROM_DATABASE=GX6338 114-Channel SPST Form-A Relay PXI Board
+
+pci:v000016E2d00006377*
+ ID_MODEL_FROM_DATABASE=GX6377 Multifunction Switching PXI Board
+
+pci:v000016E2d00006384*
+ ID_MODEL_FROM_DATABASE=GX6384 Configurable High-Density Switch Matrix PXI Board
+
+pci:v000016E2d00006616*
+ ID_MODEL_FROM_DATABASE=GX6616 6x2:16 Switch Matrix PXI Board
+
+pci:v000016E2d00006864*
+ ID_MODEL_FROM_DATABASE=GX6864 Video Switching PXI Board
+
+pci:v000016E2d00007400*
+ ID_MODEL_FROM_DATABASE=GX7400 Dual-Output Programmable Power Supply PXI Board
+
+pci:v000016E2d00007404*
+ ID_MODEL_FROM_DATABASE=GX7404 Prototyping and Power Interface PXI Board
+
+pci:v000016E2d00007777*
+ ID_MODEL_FROM_DATABASE=GX7777 Generic Interface PXI Board
+
+pci:v000016E2d00007779*
+ ID_MODEL_FROM_DATABASE=GX7779 Backplane Tester PXI Board
+
 pci:v000016E3*
  ID_VENDOR_FROM_DATABASE=European Space Agency
 
@@ -68280,7 +68457,7 @@ pci:v0000193C*
  ID_VENDOR_FROM_DATABASE=MAXIM Integrated Products
 
 pci:v0000193D*
- ID_VENDOR_FROM_DATABASE=Hangzhou H3C Technologies Co., Ltd.
+ ID_VENDOR_FROM_DATABASE=New H3C Technologies Co., Ltd.
 
 pci:v0000193F*
  ID_VENDOR_FROM_DATABASE=AHA Products Group
@@ -71453,48 +71630,90 @@ pci:v00001C5Fd0000000Esv00001C5Fsd00000B20*
 pci:v00001C5Fd0000000Esv00001C5Fsd00000B21*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 1920G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00000B25*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 1920G E1.S)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00000B27*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6630 1920G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00000B30*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 3840G AIC)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00000B31*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 3840G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00000B35*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 3840G E1.S)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00000B37*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6630 3840G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00000B40*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 7680G AIC)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00000B41*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6530 7680G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00000B47*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6630 7680G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00001331*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6531 3840G 2.5" U.2)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00001341*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6531 7680G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00001431*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6541 3840G 2.5" U.2)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00001441*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6541 7680G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B20*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 1600G AIC)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B21*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 1600G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00004B25*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 1600G E1.S)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00004B27*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6636 1600G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B30*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 3200G AIC)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B31*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 3200G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00004B35*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 3200G E1.S)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00004B37*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6636 3200G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B40*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 6400G AIC)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00004B41*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6536 6400G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00004B47*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6636 6400G 2.5" U.2)
+
 pci:v00001C5Fd0000000Esv00001C5Fsd00005331*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6537 3200G 2.5" U.2)
 
 pci:v00001C5Fd0000000Esv00001C5Fsd00005341*
  ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6537 6400G 2.5" U.2)
 
+pci:v00001C5Fd0000000Esv00001C5Fsd00005431*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6547 3200G 2.5" U.2)
+
+pci:v00001C5Fd0000000Esv00001C5Fsd00005441*
+ ID_MODEL_FROM_DATABASE=PBlaze6 6530 (NVMe SSD PBlaze6 6547 6400G 2.5" U.2)
+
 pci:v00001C5Fd0000003D*
  ID_MODEL_FROM_DATABASE=PBlaze5 920/926
 
@@ -71558,6 +71777,9 @@ pci:v00001C5Fd0000003Esv00001C5Fsd00004B51*
 pci:v00001C5Fd0000003Esv00001C5Fsd00004B61*
  ID_MODEL_FROM_DATABASE=PBlaze6 6920/6930 (NVMe SSD PBlaze6 6936 25600GB 2.5" U.3)
 
+pci:v00001C5Fd0000003F*
+ ID_MODEL_FROM_DATABASE=PBlaze7 7940/7946
+
 pci:v00001C5Fd00000540*
  ID_MODEL_FROM_DATABASE=PBlaze4 NVMe SSD
 
@@ -71709,7 +71931,7 @@ pci:v00001CC4d00001203sv00001CC4sd0000A214*
  ID_MODEL_FROM_DATABASE=NVMe SSD Controller UHXXXa series (NVMe SSD UHXXXa series U.2 6400GB)
 
 pci:v00001CC4d00001203sv00001CC4sd0000E122*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller UHXXXa series (NVMe SSD UH711a series U.2 1920G)
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller UHXXXa series (NVMe SSD UH711a series U.2 1920GB)
 
 pci:v00001CC4d00001203sv00001CC4sd0000E123*
  ID_MODEL_FROM_DATABASE=NVMe SSD Controller UHXXXa series (NVMe SSD UH711a series U.2 3840GB)
@@ -72714,7 +72936,7 @@ pci:v00001D97d00002263*
  ID_MODEL_FROM_DATABASE=SM2263EN/SM2263XT-based OEM SSD
 
 pci:v00001D9B*
- ID_VENDOR_FROM_DATABASE=Facebook, Inc.
+ ID_VENDOR_FROM_DATABASE=Meta Platforms, Inc.
 
 pci:v00001D9Bd00000010*
  ID_MODEL_FROM_DATABASE=Networking DOM Engine
@@ -73514,6 +73736,27 @@ pci:v00001DF7d00000002*
 pci:v00001DF7d00000003*
  ID_MODEL_FROM_DATABASE=alst4x
 
+pci:v00001DF8*
+ ID_VENDOR_FROM_DATABASE=V&G Information System Co.,Ltd
+
+pci:v00001DF8d0000C000*
+ ID_MODEL_FROM_DATABASE=DC NVMe SSD
+
+pci:v00001DF8d0000C000sv00001DF8sd0000C600*
+ ID_MODEL_FROM_DATABASE=DC NVMe SSD (Enterprise U.2 NVMe SSD)
+
+pci:v00001DF8d0000D000*
+ ID_MODEL_FROM_DATABASE=PC NVMe SSD
+
+pci:v00001DF8d0000D000sv00001DF8sd0000D100*
+ ID_MODEL_FROM_DATABASE=PC NVMe SSD (M.2 NVMe SSD)
+
+pci:v00001DF8d0000D000sv00001DF8sd0000D201*
+ ID_MODEL_FROM_DATABASE=PC NVMe SSD (M.2 NVMe SSD)
+
+pci:v00001DF8d0000D000sv00001DF8sd0000D600*
+ ID_MODEL_FROM_DATABASE=PC NVMe SSD (M.2 NVMe SSD)
+
 pci:v00001DFC*
  ID_VENDOR_FROM_DATABASE=JSC NT-COM
 
@@ -73713,184 +73956,223 @@ pci:v00001E3B*
  ID_VENDOR_FROM_DATABASE=DapuStor Corporation
 
 pci:v00001E3Bd00000600*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000010*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (R5102))
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000013*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (R5302))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000030*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.84TB (J5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (J5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000031*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 7.68TB (J5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 7.68TB (J5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000032*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 15.36TB (J5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 15.36TB (J5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000033*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.20TB (J5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (J5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000034*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 6.40TB (J5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 6.40TB (J5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000035*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 12.80TB (J5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 12.80TB (J5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000036*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD AIC 7.68TB (J5110))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD AIC 7.68TB (J5110))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000037*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD AIC 6.40TB (J5310))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD AIC 6.40TB (J5310))
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000038*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (J5100D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000039*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 7.68TB (J5100D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000003B*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (J5300D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000003C*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 6.40TB (J5300D))
 
 pci:v00001E3Bd00000600sv00001E3Bsd0000003E*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD AIC 3.84TB (J5110))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD AIC 3.84TB (J5110))
 
 pci:v00001E3Bd00000600sv00001E3Bsd0000003F*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD AIC 3.20TB (J5310))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD AIC 3.20TB (J5310))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000004C*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 QDP 1.92TB (J5100))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000004D*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 QDP 1.60TB (J5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000050*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.84TB (R5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (R5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000051*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 7.68TB (R5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 7.68TB (R5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000052*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 15.36TB (R5100))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 15.36TB (R5100))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000053*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.20TB (R5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (R5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000054*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 6.40TB (R5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 6.40TB (R5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000055*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 12.80TB (R5300))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 12.80TB (R5300))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000056*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.84TB (R5101))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (R5101))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000059*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.20TB (R5301))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (R5301))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000060*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.84TB (R5100D))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (R5100D))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000061*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 7.68TB (R5100D))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 7.68TB (R5100D))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000063*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.20TB (R5300D))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (R5300D))
 
 pci:v00001E3Bd00000600sv00001E3Bsd00000064*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 6.40TB (R5300D))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 6.40TB (R5300D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000066*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.84TB (R5101D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd00000069*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (R5301D))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000006C*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 1.92TB (R5101))
+
+pci:v00001E3Bd00000600sv00001E3Bsd0000006D*
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 1.60TB (J5301))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F0*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 0.40TB (X2900))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 0.40TB (X2900))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F1*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 0.80TB (X2900))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 0.80TB (X2900))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F2*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 1.60TB (X2900))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 1.60TB (X2900))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F3*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 3.20TB (X2900))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 3.20TB (X2900))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F5*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 0.40TB (X2900P))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 0.40TB (X2900P))
 
 pci:v00001E3Bd00000600sv00001E3Bsd000000F6*
- ID_MODEL_FROM_DATABASE=NVMe SSD Controller DPU600 (Enterprise NVMe SSD U.2 0.80TB (X2900P))
+ ID_MODEL_FROM_DATABASE=NVMe SSD Controller DP600 (Enterprise NVMe SSD U.2 0.80TB (X2900P))
 
 pci:v00001E3Bd00001098*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000001*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H2100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H2100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000002*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.96TB (H2200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.96TB (H2200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000004*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H2100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H2100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000005*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 1.92TB (H2200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 1.92TB (H2200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000009*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000000A*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.96TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.96TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000000C*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000000D*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 1.92TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 1.92TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000014*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 3.2TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 3.2TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000015*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 3.84TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 3.84TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000021*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 6.4TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 6.4TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000022*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 7.68TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 7.68TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000052*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.8TB (H3900))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000053*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 1.6TB (H3900))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000059*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD U.2 0.75TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD U.2 0.75TB (H3900))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000061*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H2100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H2100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000062*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.96TB (H2200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.96TB (H2200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000064*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H2100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H2100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000065*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 1.92TB (H2200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 1.92TB (H2200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000006C*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000006D*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.96TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.96TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000006F*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000070*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 1.92TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 1.92TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000007C*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 3.2TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 3.2TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000007D*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 3.84TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 3.84TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000007F*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 6.4TB (H3100))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 6.4TB (H3100))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000080*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 7.68TB (H3200))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 7.68TB (H3200))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000008A*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.8TB (H3900))
 
 pci:v00001E3Bd00001098sv00001E3Bsd0000008B*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 1.6TB (H3900))
 
 pci:v00001E3Bd00001098sv00001E3Bsd00000091*
- ID_MODEL_FROM_DATABASE=Haishen NVMe SSD (Enterprise NVMe SSD HHHL 0.75TB (H3900))
+ ID_MODEL_FROM_DATABASE=Haishen3 NVMe SSD (Enterprise NVMe SSD HHHL 0.75TB (H3900))
+
+pci:v00001E3Bd00001333*
+ ID_MODEL_FROM_DATABASE=Haishen5 NVMe SSD
 
 pci:v00001E3D*
  ID_VENDOR_FROM_DATABASE=Burlywood, Inc
@@ -73995,7 +74277,10 @@ pci:v00001E7Cd0000BCA1*
  ID_MODEL_FROM_DATABASE=AKD1000 Neural Network Coprocessor [Akida]
 
 pci:v00001E7E*
- ID_VENDOR_FROM_DATABASE=9034 Pliops-Data Processor [XDP1.0]
+ ID_VENDOR_FROM_DATABASE=Pliops
+
+pci:v00001E7Ed00009034*
+ ID_MODEL_FROM_DATABASE=Pliops Extreme Data Processor [XDP1.0]
 
 pci:v00001E7F*
  ID_VENDOR_FROM_DATABASE=Jiangsu Huacun Elec. Tech. Co., Ltd.
@@ -74439,19 +74724,19 @@ pci:v00001F3F*
  ID_VENDOR_FROM_DATABASE=3SNIC Ltd
 
 pci:v00001F3Fd00002100*
- ID_MODEL_FROM_DATABASE=SSSRAID SAS/SATA HBA
+ ID_MODEL_FROM_DATABASE=SSSHBA SAS/SATA HBA
 
 pci:v00001F3Fd00002100sv00001F3Fsd00000120*
- ID_MODEL_FROM_DATABASE=SSSRAID SAS/SATA HBA (HBA 32 Ports)
+ ID_MODEL_FROM_DATABASE=SSSHBA SAS/SATA HBA (HBA 32 Ports)
 
 pci:v00001F3Fd00002100sv00001F3Fsd00000125*
- ID_MODEL_FROM_DATABASE=SSSRAID SAS/SATA HBA (HBA 40 Ports)
+ ID_MODEL_FROM_DATABASE=SSSHBA SAS/SATA HBA (HBA 40 Ports)
 
 pci:v00001F3Fd00002100sv00001F3Fsd00000180*
- ID_MODEL_FROM_DATABASE=SSSRAID SAS/SATA HBA (HBA 16 Ports)
+ ID_MODEL_FROM_DATABASE=SSSHBA SAS/SATA HBA (HBA 16 Ports)
 
 pci:v00001F3Fd00002100sv00001F3Fsd00000185*
- ID_MODEL_FROM_DATABASE=SSSRAID SAS/SATA HBA (HBA 8 Ports)
+ ID_MODEL_FROM_DATABASE=SSSHBA SAS/SATA HBA (HBA 8 Ports)
 
 pci:v00001F3Fd00002200*
  ID_MODEL_FROM_DATABASE=SSSRAID RAID controller
@@ -83456,6 +83741,9 @@ pci:v00008086d0000159B*
 pci:v00008086d0000159Bsv00001137sd000002BE*
  ID_MODEL_FROM_DATABASE=Ethernet Controller E810-XXV for SFP (E810XXVDA2 2x25/10 GbE SFP28 PCIe NIC)
 
+pci:v00008086d0000159Bsv0000193Dsd00001085*
+ ID_MODEL_FROM_DATABASE=Ethernet Controller E810-XXV for SFP (NIC-ETH660F-3S-2P)
+
 pci:v00008086d0000159Bsv00001BD4sd00000057*
  ID_MODEL_FROM_DATABASE=Ethernet Controller E810-XXV for SFP (Ethernet Network Adapter E810-XXVAM2)
 
@@ -97238,6 +97526,15 @@ pci:v00008086d000046C1*
 pci:v00008086d000046C3*
  ID_MODEL_FROM_DATABASE=Alder Lake-UP4 GT1 [UHD Graphics]
 
+pci:v00008086d000046D0*
+ ID_MODEL_FROM_DATABASE=Alder Lake-N [UHD Graphics]
+
+pci:v00008086d000046D1*
+ ID_MODEL_FROM_DATABASE=Alder Lake-N [UHD Graphics]
+
+pci:v00008086d000046D2*
+ ID_MODEL_FROM_DATABASE=Alder Lake-N [UHD Graphics]
+
 pci:v00008086d00004905*
  ID_MODEL_FROM_DATABASE=DG1 [Iris Xe MAX Graphics]
 
@@ -98834,6 +99131,9 @@ pci:v00008086d00007ABD*
 pci:v00008086d00007ABF*
  ID_MODEL_FROM_DATABASE=Alder Lake-S PCH PCI Express Root Port #8
 
+pci:v00008086d00007AC8*
+ ID_MODEL_FROM_DATABASE=Alder Lake-S PCH PCI Express Root Port #25
+
 pci:v00008086d00007ACC*
  ID_MODEL_FROM_DATABASE=Alder Lake-S PCH Serial IO I2C Controller #0
 
@@ -102047,6 +102347,12 @@ pci:v00008086d0000A3F0*
 pci:v00008086d0000A620*
  ID_MODEL_FROM_DATABASE=6400/6402 Advanced Memory Buffer (AMB)
 
+pci:v00008086d0000A720*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [UHD Graphics]
+
+pci:v00008086d0000A721*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [UHD Graphics]
+
 pci:v00008086d0000A77F*
  ID_MODEL_FROM_DATABASE=Volume Management Device NVMe RAID Controller Intel Corporation
 
@@ -102074,6 +102380,18 @@ pci:v00008086d0000A78A*
 pci:v00008086d0000A78B*
  ID_MODEL_FROM_DATABASE=Raptor Lake-S UHD Graphics
 
+pci:v00008086d0000A7A0*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [Iris Xe Graphics]
+
+pci:v00008086d0000A7A1*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [Iris Xe Graphics]
+
+pci:v00008086d0000A7A8*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [UHD Graphics]
+
+pci:v00008086d0000A7A9*
+ ID_MODEL_FROM_DATABASE=Raptor Lake-P [UHD Graphics]
+
 pci:v00008086d0000ABC0*
  ID_MODEL_FROM_DATABASE=Omni-Path Fabric Switch Silicon 100 Series
 
diff --git a/hwdb.d/ma-large.txt b/hwdb.d/ma-large.txt
index b24195de94..feeaaaac76 100644
--- a/hwdb.d/ma-large.txt
+++ b/hwdb.d/ma-large.txt
@@ -40067,14 +40067,14 @@ A8C647     (base 16)		Extreme Networks, Inc.
 				San Jose  CA  95119
 				US
 
-28-C0-1B   (hex)		Shenzhen Skyworth Digital  Technology  CO., Ltd
-28C01B     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
+E0-28-B1   (hex)		Shenzhen Skyworth Digital  Technology  CO., Ltd
+E028B1     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
 				4F,Block A, Skyworth?Building,
 				Shenzhen  Guangdong  518057
 				CN
 
-E0-28-B1   (hex)		Shenzhen Skyworth Digital  Technology  CO., Ltd
-E028B1     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
+28-C0-1B   (hex)		Shenzhen Skyworth Digital  Technology  CO., Ltd
+28C01B     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
 				4F,Block A, Skyworth?Building,
 				Shenzhen  Guangdong  518057
 				CN
@@ -40091,6 +40091,18 @@ E028B1     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
 				Chongqing  Chongqing  401331
 				CN
 
+FC-84-A7   (hex)		Murata Manufacturing Co., Ltd.
+FC84A7     (base 16)		Murata Manufacturing Co., Ltd.
+				1-10-1, Higashikotari
+				Nagaokakyo-shi  Kyoto  617-8555
+				JP
+
+88-3F-37   (hex)		UHTEK CO., LTD.
+883F37     (base 16)		UHTEK CO., LTD.
+				403DONG 503HO, 655 PYEONGCHEON-RO
+				BUCHEONSI  KYEONGGIDO  14502
+				KR
+
 9C-FF-C2   (hex)		AVI Systems GmbH
 9CFFC2     (base 16)		AVI Systems GmbH
 				Dr. Franz Wilhelmstrae 2A
@@ -79664,6 +79676,30 @@ C8138B     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
 				Beijing  Beijing  100083
 				CN
 
+84-72-93   (hex)		Texas Instruments
+847293     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+F8-2E-0C   (hex)		Texas Instruments
+F82E0C     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+90-06-F2   (hex)		Texas Instruments
+9006F2     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+84-F1-75   (hex)		Jiangxi Xunte Intelligent Terminal Co., Ltd
+84F175     (base 16)		Jiangxi Xunte Intelligent Terminal Co., Ltd
+				16 # 1-3/F, Zhongxing Nanchang Software Industrial Park, No. 688, Aixihu North Road, Nanchang High-tech Industrial Development Zone, Nanchang, Jiangxi Province
+				Nanchang  Jiangxi  330000
+				CN
+
 84-80-94   (hex)		Meter, Inc.
 848094     (base 16)		Meter, Inc.
 				148 Townsend St
@@ -119645,6 +119681,30 @@ BCF499     (base 16)		Rockwell Automation
 				Mayfield Heights  OH  44124-6118
 				US
 
+8C-F3-E7   (hex)		solidotech
+8CF3E7     (base 16)		solidotech
+				No.1 Dongji Avenue, Jiangning District, Nanjing, Jiangsu Province, China
+				Nanjing  Jiangsu Province  211153
+				CN
+
+D4-52-C7   (hex)		Beijing L&S Lancom Platform Tech. Co., Ltd.
+D452C7     (base 16)		Beijing L&S Lancom Platform Tech. Co., Ltd.
+				901,floor 9, jelly loft, building 28, yard 9, Huinan Road, Changping District.
+				Beijing    102208
+				CN
+
+D4-E9-5E   (hex)		Texas Instruments
+D4E95E     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+10-15-C1   (hex)		Zhanzuo (Beijing) Technology Co., Ltd.
+1015C1     (base 16)		Zhanzuo (Beijing) Technology Co., Ltd.
+				Room 313-2511, 3rd Floor, Shanshui Building, No. 3, Gulou East Street, Miyun District
+				Beijing    101500
+				CN
+
 F8-D0-27   (hex)		Seiko Epson Corporation
 F8D027     (base 16)		Seiko Epson Corporation
 				2070 Kotobuki Koaka
@@ -159665,29 +159725,53 @@ F8B8B4     (base 16)		Shenzhen Skyworth Digital  Technology  CO., Ltd
 				Shenzhen  Guangdong  518057
 				CN
 
+8C-DF-2C   (hex)		vivo Mobile Communication Co., Ltd.
+8CDF2C     (base 16)		vivo Mobile Communication Co., Ltd.
+				No.1, vivo Road, Chang'an
+				Dongguan  Guangdong  523860
+				CN
+
+CC-B5-4C   (hex)		Texas Instruments
+CCB54C     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+74-A5-8C   (hex)		Texas Instruments
+74A58C     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+3C-E0-02   (hex)		Texas Instruments
+3CE002     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+98-03-8A   (hex)		Texas Instruments
+98038A     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
 F8-79-28   (hex)		zte corporation
 F87928     (base 16)		zte corporation
 				12/F.,zte R&D building ,kejinan Road,Shenzhen,P.R.China
 				shenzhen   guangdong  518057
 				CN
 
-D8-8E-D4   (hex)		eero inc.
-D88ED4     (base 16)		eero inc.
-				660 3rd Street
-				San Francisco  CA  94107
-				US
-
 B8-60-61   (hex)		China Mobile Group Device Co.,Ltd.
 B86061     (base 16)		China Mobile Group Device Co.,Ltd.
 				32 Xuanwumen West Street,Xicheng District
 				Beijing    100053
 				CN
 
-8C-DF-2C   (hex)		vivo Mobile Communication Co., Ltd.
-8CDF2C     (base 16)		vivo Mobile Communication Co., Ltd.
-				No.1, vivo Road, Chang'an
-				Dongguan  Guangdong  523860
-				CN
+D8-8E-D4   (hex)		eero inc.
+D88ED4     (base 16)		eero inc.
+				660 3rd Street
+				San Francisco  CA  94107
+				US
 
 7C-8A-E1   (hex)		COMPAL INFORMATION (KUNSHAN) CO., LTD. 
 7C8AE1     (base 16)		COMPAL INFORMATION (KUNSHAN) CO., LTD. 
@@ -160685,9 +160769,6 @@ D4BBC8     (base 16)		vivo Mobile Communication Co., Ltd.
 				DONG GUAN  GUANG DONG  523860
 				CN
 
-24-BF-74   (hex)		Private
-24BF74     (base 16)		Private
-
 CC-DC-55   (hex)		Dragonchip Limited
 CCDC55     (base 16)		Dragonchip Limited
 				Room 601-2, 6/F, IC Development Centre, No. 6 Science Park West Avenue, Hong Kong Science Park, Shatin, N.T.
@@ -199600,3 +199681,33 @@ E48EBB     (base 16)		Rockwell Automation
 				1 Tesla Road
 				Austin  TX  78725
 				US
+
+24-BF-74   (hex)		Hamamatsu Photonics K.K.
+24BF74     (base 16)		Hamamatsu Photonics K.K.
+				1126-1, Ichino-cho, Higashi-ku
+				Hamamatsu-City  Shizuoka-ken  435-8558
+				JP
+
+FC-48-C9   (hex)		Yobiiq Intelligence B.V.
+FC48C9     (base 16)		Yobiiq Intelligence B.V.
+				Bedrijfsweg 31
+				Geleen  Limburg  6163CZ
+				NL
+
+A0-66-36   (hex)		Intracom SA Telecom Solutions
+A06636     (base 16)		Intracom SA Telecom Solutions
+				19.7 klm Marcopoulo Ave
+				PEANIA  ATTIKI  19001
+				GR
+
+64-1C-10   (hex)		Texas Instruments
+641C10     (base 16)		Texas Instruments
+				12500 TI Blvd
+				Dallas  TX  75243
+				US
+
+F4-AA-D0   (hex)		OHSUNG
+F4AAD0     (base 16)		OHSUNG
+				335-4,SANHODAERO,GUMI,GYEONG BUK,KOREA
+				GUMI  GYEONG BUK  730-030
+				KR
diff --git a/hwdb.d/ma-medium.txt b/hwdb.d/ma-medium.txt
index 175d1568ba..3811711359 100644
--- a/hwdb.d/ma-medium.txt
+++ b/hwdb.d/ma-medium.txt
@@ -5546,10 +5546,16 @@ B00000-BFFFFF     (base 16)		Korea Bus Broadcasting
 				Seoul    06684
 				KR
 
-70-06-92   (hex)		Shenzhen Lingwei Technology Co., Ltd
-900000-9FFFFF     (base 16)		Shenzhen Lingwei Technology Co., Ltd
-				No. 50-1, Minqing Road, Longhua District
-				Shenzhen  Guangdong Province  518109
+70-06-92   (hex)		SWIT Electronics Co.,Ltd
+B00000-BFFFFF     (base 16)		SWIT Electronics Co.,Ltd
+				10 Hengtong Rd,Nanjing Economic Technological Development Zone
+				Nanjing  Jiangsu  210038
+				CN
+
+70-06-92   (hex)		Hangzhou Clounix Technology Limited
+600000-6FFFFF     (base 16)		Hangzhou Clounix Technology Limited
+				Floor 12, Building 2, Zichen International Center, NO.39, Jincheng Road, Xiaoshan District
+				Hangzhou  Zhejiang  311202
 				CN
 
 70-06-92   (hex)		Fusiostor Technologies Private Limited
@@ -5558,6 +5564,18 @@ B00000-BFFFFF     (base 16)		Korea Bus Broadcasting
 				MUMBAI  Maharashtra  400080
 				IN
 
+70-06-92   (hex)		Shenzhen Lingwei Technology Co., Ltd
+900000-9FFFFF     (base 16)		Shenzhen Lingwei Technology Co., Ltd
+				No. 50-1, Minqing Road, Longhua District
+				Shenzhen  Guangdong Province  518109
+				CN
+
+70-06-92   (hex)		Scud (Fujian) Electronics Co.,Ltd
+200000-2FFFFF     (base 16)		Scud (Fujian) Electronics Co.,Ltd
+				NO.98 ,EAST ROAD OF JIANGBIN ,MAWEI ZONE,FUZHOU
+				FUZHOU    350000
+				CN
+
 4C-4B-F9   (hex)		Shenzhen dingsheng technology co., LTD
 400000-4FFFFF     (base 16)		Shenzhen dingsheng technology co., LTD
 				Floor 3, building 5, kaijeda industrial zone, no.97, huaxing road, langkou community, dalang street, longhua district
@@ -10760,6 +10778,12 @@ A00000-AFFFFF     (base 16)		Munters
 				Pethch Tikva  Israel  4959376
 				IL
 
+70-06-92   (hex)		Skyware Protech Limited
+D00000-DFFFFF     (base 16)		Skyware Protech Limited
+				3806A, Cable TV Tower, 9 Hoi Sing Road, Tsuen Wan
+				Tsuen Wan    00000
+				HK
+
 20-85-93   (hex)		UNILUMIN GROUP CO.,LTD
 300000-3FFFFF     (base 16)		UNILUMIN GROUP CO.,LTD
 				No.112 Yongfu Rd.,BaoanDistrict,
@@ -21884,12 +21908,6 @@ E00000-EFFFFF     (base 16)		Saab Seaeye Ltd
 				Fareham  Hampshire  PO15 5SD
 				GB
 
-1C-A0-D3   (hex)		Private
-700000-7FFFFF     (base 16)		Private
-				Omonoia Avenue
-				Limassol  Cyprus  3045
-				CY
-
 D0-16-F0   (hex)		Top Guard Technologies
 D00000-DFFFFF     (base 16)		Top Guard Technologies
 				9200 Corporate BlvdSuite 250
@@ -21944,6 +21962,18 @@ C00000-CFFFFF     (base 16)		ScoreBird, LLC
 				Nashville  TN  37215
 				US
 
+70-06-92   (hex)		JMA Wireless
+800000-8FFFFF     (base 16)		JMA Wireless
+				140 Cortland Ave
+				Syracuse  NY  13202
+				US
+
+1C-A0-D3   (hex)		U-TX Technologies Ltd
+700000-7FFFFF     (base 16)		U-TX Technologies Ltd
+				Omonoia Avenue
+				Limassol  Cyprus  3045
+				CY
+
 20-85-93   (hex)		Great Lite International
 700000-7FFFFF     (base 16)		Great Lite International
 				11F., No.207-2, Sec. 3, Beixin Rd., Xindian Dist.,
@@ -27529,3 +27559,9 @@ E0-3C-1C   (hex)		Ocamar Technologies (Shanghai) Co.,Ltd.
 				2335 S Ellis St
 				Lakewood  CO  80228
 				US
+
+70-06-92   (hex)		CANAAN CREATIVE CO.,LTD.
+500000-5FFFFF     (base 16)		CANAAN CREATIVE CO.,LTD.
+				Building 1, Yard 1, No.81 Beiqing Road, Haidian District
+				Beijing    100144
+				CN
diff --git a/hwdb.d/ma-small.txt b/hwdb.d/ma-small.txt
index 4a5ef001eb..75c27371ea 100644
--- a/hwdb.d/ma-small.txt
+++ b/hwdb.d/ma-small.txt
@@ -128,9 +128,6 @@ C7A000-C7AFFF     (base 16)		ENTEC Electric & Electronic Co., LTD.
 				Hwaseong-city  Gyungki-do  445-894
 				KR
 
-70-B3-D5   (hex)		Private
-6F4000-6F4FFF     (base 16)		Private
-
 70-B3-D5   (hex)		Guan Show Technologe Co., Ltd.
 1FC000-1FCFFF     (base 16)		Guan Show Technologe Co., Ltd.
 				No.127, Jianguo 1st Rd., Lingya Dist.
@@ -6404,6 +6401,12 @@ CC2000-CC2FFF     (base 16)		TOYOGIKEN CO.,LTD.
 				Okaya  Nagano  3940081
 				JP
 
+70-B3-D5   (hex)		WDI Wise Device Inc.
+6F4000-6F4FFF     (base 16)		WDI Wise Device Inc.
+				135 west beaver creek rd unit 2
+				Richmond hill   Ontario  L4B 1l2
+				CA
+
 8C-1F-64   (hex)		i2s
 286000-286FFF     (base 16)		i2s
 				28 Rue Jean Perrin
@@ -12710,6 +12713,24 @@ A81000-A81FFF     (base 16)		3D perception AS
 				Hvalstad    1395
 				NO
 
+8C-1F-64   (hex)		Power Electronics Espana, S.L.
+0D8000-0D8FFF     (base 16)		Power Electronics Espana, S.L.
+				C/ Leonardo Da Vinci, 24-26
+				Paterna  Valencia  46980
+				ES
+
+8C-1F-64   (hex)		joint analytical systems GmbH
+31B000-31BFFF     (base 16)		joint analytical systems GmbH
+				Carl-Zeiss-Str. 49
+				Moers  NRW  47445
+				DE
+
+8C-1F-64   (hex)		Midlands Technical Co., Ltd.
+DD4000-DD4FFF     (base 16)		Midlands Technical Co., Ltd.
+				1-29-4 5F Takadanobaba
+				shinjyuku-ku  Tokyo  169-0075
+				JP
+
 70-B3-D5   (hex)		System West dba ICS Electronics
 E06000-E06FFF     (base 16)		System West dba ICS Electronics
 				7034 Commerce Circle Suite A
@@ -23741,12 +23762,6 @@ B14000-B14FFF     (base 16)		Pantherun Technologies Pvt Ltd
 				Bangalore  Karnataka  560038
 				IN
 
-70-B3-D5   (hex)		Liberator Pty Ltd
-561000-561FFF     (base 16)		Liberator Pty Ltd
-				265?Gilbert Street
-				Adelaide     SA 5000
-				AU
-
 70-B3-D5   (hex)		EASTERN SCIENCE & TECHNOLOGY CO., LTD
 2DF000-2DFFFF     (base 16)		EASTERN SCIENCE & TECHNOLOGY CO., LTD
 				6F.,No.963, Zhongzheng Rd, Zhonghe Dist.
@@ -25442,6 +25457,12 @@ CAB000-CABFFF     (base 16)		Spyder Controls Corp.
 				Oldenzaal  Overijssel  7577 BZ
 				NL
 
+70-B3-D5   (hex)		Liberator Pty Ltd
+561000-561FFF     (base 16)		Liberator Pty Ltd
+				265 Gilbert Street
+				Adelaide     SA 5000
+				AU
+
 70-B3-D5   (hex)		DISMUNTEL SAL
 92C000-92CFFF     (base 16)		DISMUNTEL SAL
 				Pol ind cotes
@@ -31634,12 +31655,6 @@ EDA000-EDAFFF     (base 16)		DEUTA-WERKE GmbH
 				Karlskoga    SE-691 27
 				SE
 
-8C-1F-64   (hex)		Liberator Pty Ltd
-BBC000-BBCFFF     (base 16)		Liberator Pty Ltd
-				265?Gilbert Street
-				Adelaide     SA 5000
-				AU
-
 70-B3-D5   (hex)		Medicomp, Inc
 278000-278FFF     (base 16)		Medicomp, Inc
 				600 Atlantis Rd
@@ -31651,3 +31666,15 @@ BBC000-BBCFFF     (base 16)		Liberator Pty Ltd
 				5 Chemin de Chiradie
 				Brignais    69530
 				FR
+
+8C-1F-64   (hex)		Liberator Pty Ltd
+BBC000-BBCFFF     (base 16)		Liberator Pty Ltd
+				265 Gilbert Street
+				Adelaide     SA 5000
+				AU
+
+8C-1F-64   (hex)		COMETA SAS
+E24000-E24FFF     (base 16)		COMETA SAS
+				9 rue marcel chabloz
+				Saint martin d'Hres    38400
+				FR
diff --git a/hwdb.d/pci.ids b/hwdb.d/pci.ids
index b0c8bdcc06..f5dcf90162 100644
--- a/hwdb.d/pci.ids
+++ b/hwdb.d/pci.ids
@@ -1,8 +1,8 @@
 #
 #	List of PCI ID's
 #
-#	Version: 2023.01.26
-#	Date:    2023-01-26 03:15:02
+#	Version: 2023.02.11
+#	Date:    2023-02-11 03:15:01
 #
 #	Maintained by Albert Pool, Martin Mares, and other volunteers from
 #	the PCI ID Project at https://pci-ids.ucw.cz/.
@@ -4038,7 +4038,7 @@
 		148c 3000  Radeon HD 4350 Go! Green 512MB GDDR3
 # 113-2E172001-003
 		174b 3000  Radeon HD 4350/4550 HyperMemory DDR2
-	9553  RV710/M92 [Mobility Radeon HD 4530/4570/545v]
+	9553  RV710/M92 [Mobility Radeon HD 4530/4570/5145/530v/540v/545v]
 		1025 015e  Mobility Radeon HD 4570
 		1025 017d  Mobility Radeon HD 4570
 		1025 0205  Mobility Radeon HD 4570 / 545v
@@ -4061,7 +4061,7 @@
 		17aa 2129  Mobility Radeon HD 545v
 		17aa 215b  Mobility Radeon HD 545v
 		17aa 21bb  Mobility Radeon HD 545v
-	9555  RV710/M92 [Mobility Radeon HD 4350/4550]
+	9555  RV711/M93 [Mobility Radeon HD 4350/4550/530v/540v/545v / FirePro RG220]
 		103c 1411  ProBook 4720s GPU (Mobility Radeon HD 4350)
 	9557  RV711/M93 GL [FirePro RG220]
 	955f  RV710/M92 [Mobility Radeon HD 4330]
@@ -12743,6 +12743,7 @@
 	25a7  GA107M [GeForce MX570]
 	25a9  GA107M [GeForce RTX 2050]
 	25aa  GA107M [GeForce MX570 A]
+	25ab  GA107M [GeForce RTX 3050 4GB Laptop GPU]
 	25ac  GN20-P0-R-K2 [GeForce RTX 3050 6GB Laptop GPU]
 	25ad  GA107 [GeForce RTX 2050]
 	25af  GA107 [GeForce RTX 3050 Engineering Sample]
@@ -12753,6 +12754,7 @@
 	25b9  GA107GLM [RTX A1000 Laptop GPU]
 	25ba  GA107GLM [RTX A2000 8GB Laptop GPU]
 	25bb  GA107GLM [RTX A500 Laptop GPU]
+	25bc  GA107
 	25e0  GA107BM [GeForce RTX 3050 Ti Mobile]
 	25e2  GA107BM [GeForce RTX 3050 Mobile]
 	25e5  GA107BM [GeForce RTX 3050 Mobile]
@@ -12769,17 +12771,22 @@
 	26f5  AD102GL [L40 CNX]
 	2704  AD103 [GeForce RTX 4080]
 	2717  GN21-X11 [GeForce RTX 4090 Laptop GPU]
+	2730  AD103GLM [RTX 5000 Ada Generation Laptop GPU]
 	2757  GN21-X11
 	2782  AD104 [GeForce RTX 4070 Ti]
 	2785  AD104
 	2786  AD104 [GeForce RTX 4070]
 	27a0  GN21-X9
 	27b8  AD104GL [L4]
+	27ba  AD104
+	27bb  AD104GLM [RTX 3500 Ada Generation Laptop GPU]
 	27e0  GN21-X9
 	2820  GN21-X6
+	2838  AD106GLM [RTX 3000 Ada Generation Laptop GPU]
 	2860  GN21-X6
 	28a0  GN21-X4
 	28a1  GN21-X2
+	28b8  AD107GLM [RTX 2000 Ada Generation Laptop GPU]
 	28e0  GN21-X4
 	28e1  GN21-X2
 10df  Emulex Corporation
@@ -13906,9 +13913,9 @@
 	3208  PT890 Host Bridge
 	3213  VPX/VPX2 PCI to PCI Bridge Controller
 	3218  K8T800M Host Bridge
-	3227  VT8237 ISA bridge [KT600/K8T800/K8T890 South]
+	3227  VT8237 ISA bridge [KT600/K8T800/K8T890/CN700 South]
 		1043 80ed  A7V600/K8V-X/A8V Deluxe motherboard
-		1106 3227  DFI KT600-AL / Soltek SL-B9D-FGR Motherboard
+		1106 3227  VT8237 ISA bridge
 		1458 5001  GA-7VT600 Motherboard
 		147b 1407  KV8-MAX3 motherboard
 		1849 3227  K7VT4 motherboard
@@ -19229,6 +19236,8 @@
 144d  Samsung Electronics Co Ltd
 	1600  Apple PCIe SSD
 	a544  Exynos 8890 PCIe Root Complex
+	a575  Exynos 7420 PCIe Root Complex
+	a5e3  Exynos 5433 PCIe Root Complex
 	a800  XP941 PCIe SSD
 	a802  NVMe SSD Controller SM951/PM951
 		144d a801  PM963 2.5" NVMe PCIe SSD
@@ -22120,12 +22129,62 @@
 	1034  GX1034 Certification and Verification Module PXI Board
 	1100  GX1100 Arbitrary Waveform and Function Generator PXI Board
 	1120  GX1120 Arbitrary Waveform and Function Generator PXI Board
+	1164  GX1164 Multi-Channel Programmable Resistor PXI Board
 	1632  GX1632e 32-Channel Arbitrary Analog Output PXIe Board
+	1648  GX1648 64-Channel Analog Output PXI Board
+	1649  GX1649 64-Channel Arbitrary Analog Output PXI Board
+	1838  GX1838 Precision DC Source PXI Board
 	2065  GX2065 Digital Multimeter PXI Board
 	2200  GTX2200 High Resolution Universal Time Interval Counter PXI Board
 	2201  GC2200 High Resolution Universal Time Interval Counter PCI Board
+	3104  GX3104 4-Channel 20V, 250mA Per Channel SMU PXI Board
+	3108  GX3108 4-Channel Source Measure Unit PXI Board
+	3116  GX3116e 16-Channel DUT Power Supply PXIe Board
+	3348  GX3348 Multi-Channel DC Source and Switch Matrix Board
+	3500  GX3500 Digital I/O FLEX FPGA Board
+	3700  GX3700e Digital I/O FLEX FPGA PXIe Board
+	3701  GX3700 Digital I/O FLEX FPGA PXI Board
+	3800  GX3800e Digital I/O FLEX FPGA PXIe Board
+	4943  GX4943 HellFire/Longbow Serial Communication Board
 	5050  GC5050 Dynamic Digital I/O with Algorithmic Sequencer PCI Board
+	5055  GX5055 Dynamic Digital I/O with Pin Electronics PXI Board
+	5083  GX5083 Dynamic Digital I/O with Algorithmic Sequencer PXI Board
+	5106  GX5106 High-Speed Digital I/O Cycle Timing (Master) Module PXI Board
+	5151  GX5150/1 High Speed Digital I/O (50/100MHz) PXI Board
+	5152  GX5152/3 DSR High Speed Digital Stimulus/Response PXI Board
+	5250  GX5250 Digital I/O Streaming PXI Board
+	5280  GX5280 High-Speed Digital I/O PXI Board
+	5290  GX5290 High-Speed Dynamic Digital I/O PXI Board
+	5294  GX5294 High-Speed Digital I/O PXI Board
+	5295  GX5295 Digital I/O with Pin Electronics and PMU PXI Board
+	5296  GX5296 Digital I/O with Pin Electronics and PMU PXI Board
+	5550  GX5050 Dynamic Digital I/O with Algorithmic Sequencer PXI Board
 	5641  GX5641/2 Bi-directional Differential-TTL I/O PXI Board
+	5731  GX5731 Digital I/O with 128 TTL Channels and 3x32 Customizable Channels PXI Board
+	5732  GX5732 224-Channel Static Digital I/O PXI Board
+	5733  GX5733 Digital I/O with 96 TTL Channels and 32 Customizable Channels PXI Board
+	5960  GX5961/4 Digital I/O Timing/Sync with Pin Electronics and PMU PXI Board
+	6021  GX6021 20-Channel RF Multiplexer PXI Board
+	6062  GX6062 60-Channel RF Multiplexer PXI Board
+	6115  GX6115 15-Channel High Current SPDT Form-C Relay PXI Board
+	6125  GX6125 25-Channel High-Density SPDT Form-C Relay PXI Board
+	6138  GX6138 38-Channel SPST Form-A Relay PXI Board
+	6188  GX6188 104-Channel 8x8 Multiplexer/Matrix PXI Board
+	6192  GX6192 16, 16x2 High Frequency, High Density Multiplexer PXI Board
+	6196  GX6196 96-Channel DPST Form-A Relay Carrier PXI Board
+	6256  GX6256 16, 16x2 Low Frequency, High Density Multiplexer PXI Board
+	6264  GX6264 128-Channel Scanner/Multiplexer Board
+	6315  GX6315 45-Channel High Current SPDT Form-C Relay Board
+	6325  GX6325 75-Channel High-Density SPDT Form-C Relay Board
+	6338  GX6338 114-Channel SPST Form-A Relay PXI Board
+	6377  GX6377 Multifunction Switching PXI Board
+	6384  GX6384 Configurable High-Density Switch Matrix PXI Board
+	6616  GX6616 6x2:16 Switch Matrix PXI Board
+	6864  GX6864 Video Switching PXI Board
+	7400  GX7400 Dual-Output Programmable Power Supply PXI Board
+	7404  GX7404 Prototyping and Power Interface PXI Board
+	7777  GX7777 Generic Interface PXI Board
+	7779  GX7779 Backplane Tester PXI Board
 16e3  European Space Agency
 	1e0f  LEON2FT Processor
 16e5  Intellon Corp.
@@ -23164,7 +23223,7 @@
 	000c  Qualcomm MSM6275 UMTS chip
 1932  DiBcom
 193c  MAXIM Integrated Products
-193d  Hangzhou H3C Technologies Co., Ltd.
+193d  New H3C Technologies Co., Ltd.
 193f  AHA Products Group
 	0001  AHA36x-PCIX
 	0360  AHA360-PCIe
@@ -24332,20 +24391,34 @@
 	000e  PBlaze6 6530
 		1c5f 0b20  NVMe SSD PBlaze6 6530 1920G AIC
 		1c5f 0b21  NVMe SSD PBlaze6 6530 1920G 2.5" U.2
+		1c5f 0b25  NVMe SSD PBlaze6 6530 1920G E1.S
+		1c5f 0b27  NVMe SSD PBlaze6 6630 1920G 2.5" U.2
 		1c5f 0b30  NVMe SSD PBlaze6 6530 3840G AIC
 		1c5f 0b31  NVMe SSD PBlaze6 6530 3840G 2.5" U.2
+		1c5f 0b35  NVMe SSD PBlaze6 6530 3840G E1.S
+		1c5f 0b37  NVMe SSD PBlaze6 6630 3840G 2.5" U.2
 		1c5f 0b40  NVMe SSD PBlaze6 6530 7680G AIC
 		1c5f 0b41  NVMe SSD PBlaze6 6530 7680G 2.5" U.2
+		1c5f 0b47  NVMe SSD PBlaze6 6630 7680G 2.5" U.2
 		1c5f 1331  NVMe SSD PBlaze6 6531 3840G 2.5" U.2
 		1c5f 1341  NVMe SSD PBlaze6 6531 7680G 2.5" U.2
+		1c5f 1431  NVMe SSD PBlaze6 6541 3840G 2.5" U.2
+		1c5f 1441  NVMe SSD PBlaze6 6541 7680G 2.5" U.2
 		1c5f 4b20  NVMe SSD PBlaze6 6536 1600G AIC
 		1c5f 4b21  NVMe SSD PBlaze6 6536 1600G 2.5" U.2
+		1c5f 4b25  NVMe SSD PBlaze6 6536 1600G E1.S
+		1c5f 4b27  NVMe SSD PBlaze6 6636 1600G 2.5" U.2
 		1c5f 4b30  NVMe SSD PBlaze6 6536 3200G AIC
 		1c5f 4b31  NVMe SSD PBlaze6 6536 3200G 2.5" U.2
+		1c5f 4b35  NVMe SSD PBlaze6 6536 3200G E1.S
+		1c5f 4b37  NVMe SSD PBlaze6 6636 3200G 2.5" U.2
 		1c5f 4b40  NVMe SSD PBlaze6 6536 6400G AIC
 		1c5f 4b41  NVMe SSD PBlaze6 6536 6400G 2.5" U.2
+		1c5f 4b47  NVMe SSD PBlaze6 6636 6400G 2.5" U.2
 		1c5f 5331  NVMe SSD PBlaze6 6537 3200G 2.5" U.2
 		1c5f 5341  NVMe SSD PBlaze6 6537 6400G 2.5" U.2
+		1c5f 5431  NVMe SSD PBlaze6 6547 3200G 2.5" U.2
+		1c5f 5441  NVMe SSD PBlaze6 6547 6400G 2.5" U.2
 	003d  PBlaze5 920/926
 		1c5f 0a30  NVMe SSD PBlaze5 920 3840G AIC
 		1c5f 0a31  NVMe SSD PBlaze5 920 3840G 2.5" U.2
@@ -24367,6 +24440,8 @@
 		1c5f 4b41  NVMe SSD PBlaze6 6936 6400GB 2.5" U.3
 		1c5f 4b51  NVMe SSD PBlaze6 6936 12800GB 2.5" U.3
 		1c5f 4b61  NVMe SSD PBlaze6 6936 25600GB 2.5" U.3
+# Gen5 NVMe SSD
+	003f  PBlaze7 7940/7946
 	0540  PBlaze4 NVMe SSD
 	0550  PBlaze5 700/900
 	0555  PBlaze5 510/516
@@ -24425,7 +24500,7 @@
 		1cc4 a212  NVMe SSD UHXXXa series U.2 1600GB
 		1cc4 a213  NVMe SSD UHXXXa series U.2 3200GB
 		1cc4 a214  NVMe SSD UHXXXa series U.2 6400GB
-		1cc4 e122  NVMe SSD UH711a series U.2 1920G
+		1cc4 e122  NVMe SSD UH711a series U.2 1920GB
 		1cc4 e123  NVMe SSD UH711a series U.2 3840GB
 		1cc4 e124  NVMe SSD UH711a series U.2 7680GB
 	17ab  NVMe 256G SSD device
@@ -24767,7 +24842,8 @@
 	0002  Colossus GC1 [S1]
 1d97  Shenzhen Longsys Electronics Co., Ltd.
 	2263  SM2263EN/SM2263XT-based OEM SSD
-1d9b  Facebook, Inc.
+# nee Facebook, Inc.
+1d9b  Meta Platforms, Inc.
 	0010  Networking DOM Engine
 	0011  IO Bridge
 1da1  Teko Telecom S.r.l.
@@ -25057,6 +25133,13 @@
 	0001  ml605
 	0002  alst4
 	0003  alst4x
+1df8  V&G Information System Co.,Ltd
+	c000  DC NVMe SSD
+		1df8 c600  Enterprise U.2 NVMe SSD
+	d000  PC NVMe SSD
+		1df8 d100  M.2 NVMe SSD
+		1df8 d201  M.2 NVMe SSD
+		1df8 d600  M.2 NVMe SSD
 1dfc  JSC NT-COM
 	1181  TDM 8 Port E1/T1/J1 Adapter
 1e0f  KIOXIA Corporation
@@ -25128,7 +25211,9 @@
 # https://www.medion.com/
 1e39  MEDION AG
 1e3b  DapuStor Corporation
-	0600  NVMe SSD Controller DPU600
+	0600  NVMe SSD Controller DP600
+		1e3b 0010  Enterprise NVMe SSD U.2 3.84TB (R5102)
+		1e3b 0013  Enterprise NVMe SSD U.2 3.20TB (R5302)
 		1e3b 0030  Enterprise NVMe SSD U.2 3.84TB (J5100)
 		1e3b 0031  Enterprise NVMe SSD U.2 7.68TB (J5100)
 		1e3b 0032  Enterprise NVMe SSD U.2 15.36TB (J5100)
@@ -25137,8 +25222,14 @@
 		1e3b 0035  Enterprise NVMe SSD U.2 12.80TB (J5300)
 		1e3b 0036  Enterprise NVMe SSD AIC 7.68TB (J5110)
 		1e3b 0037  Enterprise NVMe SSD AIC 6.40TB (J5310)
+		1e3b 0038  Enterprise NVMe SSD U.2 3.84TB (J5100D)
+		1e3b 0039  Enterprise NVMe SSD U.2 7.68TB (J5100D)
+		1e3b 003b  Enterprise NVMe SSD U.2 3.20TB (J5300D)
+		1e3b 003c  Enterprise NVMe SSD U.2 6.40TB (J5300D)
 		1e3b 003e  Enterprise NVMe SSD AIC 3.84TB (J5110)
 		1e3b 003f  Enterprise NVMe SSD AIC 3.20TB (J5310)
+		1e3b 004c  Enterprise NVMe SSD U.2 QDP 1.92TB (J5100)
+		1e3b 004d  Enterprise NVMe SSD U.2 QDP 1.60TB (J5300)
 		1e3b 0050  Enterprise NVMe SSD U.2 3.84TB (R5100)
 		1e3b 0051  Enterprise NVMe SSD U.2 7.68TB (R5100)
 		1e3b 0052  Enterprise NVMe SSD U.2 15.36TB (R5100)
@@ -25151,13 +25242,17 @@
 		1e3b 0061  Enterprise NVMe SSD U.2 7.68TB (R5100D)
 		1e3b 0063  Enterprise NVMe SSD U.2 3.20TB (R5300D)
 		1e3b 0064  Enterprise NVMe SSD U.2 6.40TB (R5300D)
+		1e3b 0066  Enterprise NVMe SSD U.2 3.84TB (R5101D)
+		1e3b 0069  Enterprise NVMe SSD U.2 3.20TB (R5301D)
+		1e3b 006c  Enterprise NVMe SSD U.2 1.92TB (R5101)
+		1e3b 006d  Enterprise NVMe SSD U.2 1.60TB (J5301)
 		1e3b 00f0  Enterprise NVMe SSD U.2 0.40TB (X2900)
 		1e3b 00f1  Enterprise NVMe SSD U.2 0.80TB (X2900)
 		1e3b 00f2  Enterprise NVMe SSD U.2 1.60TB (X2900)
 		1e3b 00f3  Enterprise NVMe SSD U.2 3.20TB (X2900)
 		1e3b 00f5  Enterprise NVMe SSD U.2 0.40TB (X2900P)
 		1e3b 00f6  Enterprise NVMe SSD U.2 0.80TB (X2900P)
-	1098  Haishen NVMe SSD
+	1098  Haishen3 NVMe SSD
 		1e3b 0001  Enterprise NVMe SSD U.2 0.8TB (H2100)
 		1e3b 0002  Enterprise NVMe SSD U.2 0.96TB (H2200)
 		1e3b 0004  Enterprise NVMe SSD U.2 1.6TB (H2100)
@@ -25188,6 +25283,7 @@
 		1e3b 008a  Enterprise NVMe SSD HHHL 0.8TB (H3900)
 		1e3b 008b  Enterprise NVMe SSD HHHL 1.6TB (H3900)
 		1e3b 0091  Enterprise NVMe SSD HHHL 0.75TB (H3900)
+	1333  Haishen5 NVMe SSD
 1e3d  Burlywood, Inc
 1e44  Valve Software
 1e49  Yangtze Memory Technologies Co.,Ltd
@@ -25224,7 +25320,8 @@
 1e7b  Dataland
 1e7c  Brainchip Inc
 	bca1  AKD1000 Neural Network Coprocessor [Akida]
-1e7e  9034 Pliops-Data Processor [XDP1.0]
+1e7e  Pliops
+	9034  Pliops Extreme Data Processor [XDP1.0]
 1e7f  Jiangsu Huacun Elec. Tech. Co., Ltd.
 1e81  Ramaxel Technology(Shenzhen) Limited
 	1203  NVMe SSD Controller UHXXXa series
@@ -25373,7 +25470,7 @@
 		1f2f 6116  KM560 U.2 3.84TB NVMe SSD
 		1f2f 6118  KM560 U.2 7.68TB NVMe SSD
 1f3f  3SNIC Ltd
-	2100  SSSRAID SAS/SATA HBA
+	2100  SSSHBA SAS/SATA HBA
 		1f3f 0120  HBA 32 Ports
 		1f3f 0125  HBA 40 Ports
 		1f3f 0180  HBA 16 Ports
@@ -28540,6 +28637,8 @@
 	159a  Ethernet Controller E810-XXV for QSFP
 	159b  Ethernet Controller E810-XXV for SFP
 		1137 02be  E810XXVDA2 2x25/10 GbE SFP28 PCIe NIC
+# NIC-ETH660F-3S-2P 2x25GbE SFP28 Network Adapter for OCP 3.0
+		193d 1085  NIC-ETH660F-3S-2P
 		1bd4 0057  Ethernet Network Adapter E810-XXVAM2
 		1bd4 0058  Ethernet Network Adapter E810-XXVAM2 for OCP 3.0
 		1bd4 006e  Ethernet Network Adapter E810-XXVAM2 for BD
@@ -33147,6 +33246,9 @@
 	46c0  AlderLake-M GT1
 	46c1  AlderLake-M [Iris Xe Graphics]
 	46c3  Alder Lake-UP4 GT1 [UHD Graphics]
+	46d0  Alder Lake-N [UHD Graphics]
+	46d1  Alder Lake-N [UHD Graphics]
+	46d2  Alder Lake-N [UHD Graphics]
 	4905  DG1 [Iris Xe MAX Graphics]
 	4906  DG1 [Iris Xe Pod]
 	4907  SG1 [Server GPU SG-18M]
@@ -33680,6 +33782,8 @@
 	7abc  Alder Lake-S PCH PCI Express Root Port #5
 	7abd  Alder Lake-S PCH PCI Express Root Port #6
 	7abf  Alder Lake-S PCH PCI Express Root Port #8
+# As documented in Chipset datasheet
+	7ac8  Alder Lake-S PCH PCI Express Root Port #25
 	7acc  Alder Lake-S PCH Serial IO I2C Controller #0
 	7acd  Alder Lake-S PCH Serial IO I2C Controller #1
 	7ace  Alder Lake-S PCH Serial IO I2C Controller #2
@@ -34752,6 +34856,8 @@
 	a3eb  Comet Lake PCI Express Root Port #21
 	a3f0  Comet Lake PCH-V cAVS
 	a620  6400/6402 Advanced Memory Buffer (AMB)
+	a720  Raptor Lake-P [UHD Graphics]
+	a721  Raptor Lake-P [UHD Graphics]
 	a77f  Volume Management Device NVMe RAID Controller Intel Corporation
 	a780  Raptor Lake-S GT1 [UHD Graphics 770]
 	a781  Raptor Lake-S UHD Graphics
@@ -34761,6 +34867,10 @@
 	a789  Raptor Lake-S UHD Graphics
 	a78a  Raptor Lake-S UHD Graphics
 	a78b  Raptor Lake-S UHD Graphics
+	a7a0  Raptor Lake-P [Iris Xe Graphics]
+	a7a1  Raptor Lake-P [Iris Xe Graphics]
+	a7a8  Raptor Lake-P [UHD Graphics]
+	a7a9  Raptor Lake-P [UHD Graphics]
 	abc0  Omni-Path Fabric Switch Silicon 100 Series
 	ad0b  Volume Management Device NVMe RAID Controller Intel Corporation
 	b152  21152 PCI-to-PCI Bridge
